Code/GraphMol/Descriptors/GETAWAY.cpp

```diff
@@ -0,0 +1,804 @@
+
+   for (int i=0;i<8;i++){
+    res.push_back(Rk[3][i]);
+   }
+  res.push_back(RTe);
+
+  for (int i=0;i<8;i++){
+    res.push_back(Rp[3][i]);
+   }
+  res.push_back(RTMe);
+
+   for (int i=0;i<8;i++){
+    res.push_back(Rk[4][i]);
+   }
+  res.push_back(RTp);
+
+  for (int i=0;i<8;i++){
+    res.push_back(Rp[4][i]);
+   }
+  res.push_back(RTMp);
+
+   for (int i=0;i<8;i++){
+    res.push_back(Rk[5][i]);
+   }
+  res.push_back(RTi);
+
+  for (int i=0;i<8;i++){
+    res.push_back(Rp[5][i]);
+   }
+  res.push_back(RTMi);
+
+  // wrong vs Dragon
+   for (int i=0;i<8;i++){
+    res.push_back(Rk[6][i]);
+   }
+   // wrong vs Dragon
+  res.push_back(RTs);
+  // wrong vs Dragon
+  for (int i=0;i<8;i++){
+    res.push_back(Rp[6][i]);
+   }
+   // wrong vs Dragon
+  res.push_back(RTMs);
+
+  return res;
+}
+
+
+std::vector<double> GetGETAWAY(const Conformer &conf, double Vpoints[], MatrixXd DM, MatrixXd ADJ, std::vector<int> Heavylist){
+
+    std::vector<std::string> GETAWAYNAMES={"ITH","ISH","HIC","HGM","H0u","H1u","H2u","H3u","H4u","H5u","H6u","H7u","H8u","HTu",
+    "HATS0u","HATS1u","HATS2u","HATS3u","HATS4u","HATS5u","HATS6u","HATS7u","HATS8u","HATSu","H0m","H1m","H2m","H3m","H4m","H5m",
+    "H6m","H7m","H8m","HTm","HATS0m","HATS1m","HATS2m","HATS3m","HATS4m","HATS5m","HATS6m","HATS7m","HATS8m","HATSm","H0v","H1v",
+    "H2v","H3v","H4v","H5v","H6v","H7v","H8v","HTv","HATS0v","HATS1v","HATS2v","HATS3v","HATS4v","HATS5v","HATS6v","HATS7v","HATS8v",
+    "HATSv","H0e","H1e","H2e","H3e","H4e","H5e","H6e","H7e","H8e","HTe","HATS0e","HATS1e","HATS2e","HATS3e","HATS4e","HATS5e","HATS6e",
+    "HATS7e","HATS8e","HATSe","H0p","H1p","H2p","H3p","H4p","H5p","H6p","H7p","H8p","HTp","HATS0p","HATS1p","HATS2p","HATS3p","HATS4p",
+    "HATS5p","HATS6p","HATS7p","HATS8p","HATSp","H0i","H1i","H2i","H3i","H4i","H5i","H6i","H7i","H8i","HTi","HATS0i","HATS1i","HATS2i",
+    "HATS3i","HATS4i","HATS5i","HATS6i","HATS7i","HATS8i","HATSi","H0s","H1s","H2s","H3s","H4s","H5s","H6s","H7s","H8s","HTs","HATS0s",
+    "HATS1s","HATS2s","HATS3s","HATS4s","HATS5s","HATS6s","HATS7s","HATS8s","HATSs","RCON","RARS","REIG","R1u","R2u","R3u","R4u","R5u",
+    "R6u","R7u","R8u","RTu","R1u+","R2u+","R3u+","R4u+","R5u+","R6u+","R7u+","R8u+","RTu+","R1m","R2m","R3m","R4m","R5m","R6m","R7m",
+    "R8m","RTm","R1m+","R2m+","R3m+","R4m+","R5m+","R6m+","R7m+","R8m+","RTm+","R1v","R2v","R3v","R4v","R5v","R6v","R7v","R8v","RTv",
+    "R1v+","R2v+","R3v+","R4v+","R5v+","R6v+","R7v+","R8v+","RTv+","R1e","R2e","R3e","R4e","R5e","R6e","R7e","R8e","RTe","R1e+","R2e+",
+    "R3e+","R4e+","R5e+","R6e+","R7e+","R8e+","RTe+","R1p","R2p","R3p","R4p","R5p","R6p","R7p","R8p","RTp","R1p+","R2p+","R3p+","R4p+",
+    "R5p+","R6p+","R7p+","R8p+","RTp+","R1i","R2i","R3i","R4i","R5i","R6i","R7i","R8i","RTi","R1i+","R2i+","R3i+","R4i+","R5i+","R6i+",
+    "R7i+","R8i+","RTi+","R1s","R2s","R3s","R4s","R5s","R6s","R7s","R8s","RTs","R1s+","R2s+","R3s+","R4s+","R5s+","R6s+","R7s+","R8s+","RTs+"};
```

Comment:
no it's again a remining of the naming of those variables... I don't know how to generate them outside the computationnal process. That would be nice to do this independently of the code ... but if we decide to add new descriptors we need to have a easy way to update those names